### PR TITLE
Updates due to org change to Astralway

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: conf/webindex-env.sh.example [hunk 1]
```diff
@@ -16,11 +16,18 @@
 # export statement to use the correct directory.  Remove the test statement 
 # to override any previously set environment.
 
+# Hadoop
 test -z "$HADOOP_PREFIX" && export HADOOP_PREFIX=/path/to/hadoop
 test -z "$HADOOP_CONF_DIR" && export HADOOP_CONF_DIR=/path/to/hadoop/etc/hadoop
+export HADOOP_VERSION=2.7.2
```
Comment (keith-turner) [inline conf/webindex-env.sh.example]: May be useful to commend thats it important to use version installed on system

File: bin/impl/base.sh [hunk 2]
```diff
@@ -38,5 +38,5 @@ export WI_DATA_DEP_JAR=$WI_HOME/modules/data/target/webindex-data-0.0.1-SNAPSHOT
 if [ ! -f $WI_DATA_DEP_JAR ]; then
   echo "Building $WI_DATA_DEP_JAR"
   cd $WI_HOME
-  mvn clean package -DskipTests
+  mvn clean package -DskipTests -Dspark.version=$SPARK_VERSION -Dhadoop.version=$HADOOP_VERSION
```
Comment (keith-turner) [inline bin/impl/base.sh]: Could have a sanity check to ensure these are set.  Like this script checks that some other vars are set.

**Comment by keith-turner** [comment]
+1

Was able to run `mvn verify`.  Also all of the following came back empty.

```bash
$ find bin -type f | xargs grep "io.fluo"
$ find docs -type f | xargs grep "io.fluo"
$ find modules -type f | xargs grep "io.fluo"
$ find contrib -type f | xargs grep "io.fluo"
$ find conf -type f | xargs grep "io.fluo"
```

File: bin/impl/base.sh (hunk 2 above)
Comment (mikewalch) [inline bin/impl/base.sh]: Added checks in 9d6b3183d1d7f

File: conf/webindex-env.sh.example (hunk 1 above)
Comment (mikewalch) [inline conf/webindex-env.sh.example]: Added checks in 9d6b3183d1d7f

**Comment by mikewalch** [comment]
There were some other minor changes made in 9d6b3183d1d7f.  Let me if it looks good.

**Pull request merged**